lib/fs.js

```diff
@@ -520,7 +520,6 @@ function tryStatSync(fd, isUserFd) {
   } finally {
```

Comment:
Of course but I ran the benchmarks with changing these to try catch and it did not show a changed numbers. Therefore I think it's mainly churn to change it.